build/static/boot/token.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+file_token="$CLUSTER_PATH/cluster_token.txt"
+
+if [[ -n "$DST_TOKEN" ]] && [[ ! -f "$file_token" ]]; then
```

Comment:
I thought the same but I liked more `DST_TOKEN`. But if you prefer just `TOKEN` I can change it.